hnn_core/tests/test_cell_response.py

```diff
@@ -25,8 +25,41 @@ def test_cell_response(tmpdir):
                                  spike_types=spike_types,
                                  times=sim_times)
     cell_response.plot_spikes_hist(show=False)
-    cell_response.write(tmpdir.join('spk_%d.txt'))
+
+    # Testing writing using txt files
+    with pytest.warns(DeprecationWarning,
+                      match="Writing cell response to txt files is "
+                      "deprecated."):
+        cell_response.write(tmpdir.join('spk_%d.txt'))
+
+    # Testing reading from txt files
     assert cell_response == read_spikes(tmpdir.join('spk_*.txt'))
+
+    # Testing write using hdf5
+    cell_response.write(tmpdir.join('spk.hdf5'))
+
+    # Testing when overwrite is False and same filename is used
+    with pytest.raises(FileExistsError,
+                       match="File already exists at path %s. Rename the "
+                             "file or set overwrite=True."
+                             % (tmpdir.join('spk.hdf5'),)):
```

Comment:
```suggestion
                       match="File already exists at path "):
```

You don't need to be exact with the `match` ... better to keep it a little concise and let the tests remain readable.

Reply:
make all the other tests a bit more concise following this example